fastlane/Fastfile

```diff
@@ -78,7 +78,7 @@ platform :android do
   lane :test do |options|
     flavor=options[:flavor].capitalize
 	type=options[:type].capitalize
-    gradle(task: "test#{flavor}#{type}UnitTest")
+    gradle(task: "test#{flavor}#{type}UnitTest --info")
```

Comment:
This is to find out why some tests are failing.  This must removed before review or merge.